### DISCUSSION TITLE
記事への「いいね」は取り消すことができますか？

### DISCUSSION THREAD

**Discussion by tsubolabo** [increments/qiita-discussions#337] [ヘルプ]
記事への「いいね」ボタンは次のような場合、押し間違えることもあります。

1.  　揺れるバス・電子の中で誤タッチ
2.  　「いいねしたユーザー一覧」を見るつもりで、誤って「いいね」ボタンを押してしまう。
3.  　その他、マウスを持つ手が滑る、震える（老人限定）。

このような場合、取り消す手段はあるのですか？

　私自身は上記2.の経験はありますが、全く同感できない記事にまで押したことはないので取り消すつもりはありません。しかし、私の記事に「いいね」していただいた中に気になる１件があります。

　記事の中で「いいね」しないように牽制しているのに、それでも「いいね」してくれた人がいます。ひょっとして、誤タッチした結果が消せなくて迷惑が掛かっているのではないか？　と気になっています。
　[技術者倫理が基本だ！　反論は許さない！](https://qiita.com/tsubolabo/items/7daffba531ae02641c31)

**Comment by munierujp** [comment]
僕が過去にいいねを押した記事について、ストックを見直す中で（※僕は基本的にいいねとストックをセットで使用しています）あとから解除することをまれにおこなっているので、取り消すことはできると思います。

**Reply by tsubolabo** [reply]
[munierujp](https://github.com/munierujp) さん
コメントありがとうございます。

>あとから解除することをまれにおこなっているので、取り消すことはできると思います。

そうでしたか。それらしいボタンがどこにも見つからなかったので、できないと思っていました。もう一度、捜してみます。

しかし、他の方からいただいた「いいね」まで消すには、超法規的措置が要りそうですね。

**Reply by munierujp** [reply]
TwitterのいいねやGitHubのスターなどと同様に、いいねボタンをもう一度押すことで取り消すことができます。

**Reply by tsubolabo** [reply]
[munierujp](https://github.com/munierujp) さん
ご教示ありがとうございます。
>いいねボタンをもう一度押すことで取り消すことができます。

え～っ！　そんなに簡単に？
試してみました。「いいね」数が少ない記事でやったのでは、ぬか喜びさせることになって申し訳ないので、「いいね」800回越えの人気記事で試しました。確かにお教えいただいた通りでした。

Twitterはしたことがないし、GitHubにはここのDiscussions専用に加入しているだけなので、この常識には全く気づきませんでした。

実は、Discussionsの「↑」マークは使い方も分からないまま、ハート、目玉、ロケット等々のマークも使い分けを理解しないまま、適当に操作しているだけでその適否も良く分かっていませんでした。勉強して出直してきます。

それに、こんなに簡単な取り消し方があるのなら、いただいた「いいね」も不本意にそのままになっている訳ではなさそうです。有難く頂戴しておいて良さそうな気がしてきました。

**Reply by tanaka-asami415** [reply]
@munierujp 
ありがとうございます🙇

@tsubolabo 
問題が解決されたようでよかったです！
